pulp_rpm/tests/functional/Makefile

```diff
@@ -0,0 +1,25 @@
+CPU_COUNT=$(shell python3 -c "from multiprocessing import cpu_count; print(cpu_count())")
+
+help:
+	@echo "Please use \`make <target>' where <target> is one of:"
+	@echo "  help            to show this message"
+	@echo "  install-linters to install linters."
+	@echo "  lint            to run all linters"
+	@echo "  lint-flake8     to run the flake8 linter"
+	@echo "  lint-pylint     to run the pylint linter"
+
+# It is assumed that a virtualenv is available to install the required linters.
+install-linters:
+	pip install -q flake8 flake8-docstrings flake8-quotes pydocstyle pylint
```

Comment:
Install directly from ```test_requirements.txt``` please.  Use ```pip install -r test_requirements.txt``` and add pydocstyle to the list

Reply:
I did not want to mess with the existent requirements. That is why I did like this. If I add more linters `test_requirements.txt` I am assuming that this will enforce other rules. Not sure if you want this.

Reply:
That's probably the case if you were to add more flake8 packages, but I don't think adding pydocstyle would affect anything unless we called it explicitly in Travis

That reasoning is good enough for me though -- we can keep it this way if you prefer it.